geometric/optimize.py

```diff
@@ -418,11 +435,203 @@ def SortedEigenvalues(self, H):
         else:
             logger.info("Hessian Eigenvalues: " + ' '.join("%.5e" % i for i in Eig) + '\n')
         return Eig
-        
-    def step(self):
+
+    def find_lambda(self, Lambda, Heig, Hvec, g_M, p_M):
+        """
+        Equation 26 from Gonzalez & Schlegel (1990)
+        """
+        a = Heig * np.dot(p_M, Hvec) - np.dot(g_M, Hvec)
+        b = Heig - Lambda
+        c = (a / b) ** 2
+        return np.sum(c) - (0.5 * self.IRC_stepsize) ** 2
+
+    def guess_g(self, g, H, disp):
         """
-        Perform one step of the optimization.
+        Guess the gradient using the quadratic expansion
         """
+        g_new = g + np.dot(H, disp)
+        return g_new
+
+    def step(self):
```

Comment:
I can't easily see whether `step()` has changed very much because the diff puts the function at the bottom, but it seems it hasn't changed much.  Could you comment on it, or rearrange the ordering of functions so that the diff for `step()` is shown more clearly?

Reply:
`step()` for the optimization did not change. I added a [comment](https://github.com/leeping/geomeTRIC/pull/197/files#diff-5da1085de6679e8eb0ed65e4cf06827a071fa48dff306e1645ce2f19779728bbR583).